src/main/resources/SchemaForGrammars.xsd

```diff
@@ -242,7 +243,8 @@
                             <xs:element name="attributeNamespaceID" type="xs:unsignedInt"/>
                             <xs:element name="attributeLocalNameID" type="xs:unsignedInt"/>
                             <!-- attribute datatype -->
-                            <xs:element name="attributeDatatypeID" type="xs:unsignedInt"/>
+                            <xs:element name="attributeDatatypeID" type="xs:unsignedInt"

```

Comment:
I think this is the only place is needed.
It only happens if there are tow or more attributes with the same name and different type... in elementFragmentGrammar (see https://www.w3.org/TR/exi/#informedElementFragGrammar)